warzone/Cards.cpp

```diff
@@ -526,6 +526,9 @@ namespace Cards {
 	 * 
 	 * \param deck
 	 */
+	Hand::Hand() {
+
+	}
```

Comment:
should be moved up 4 lines so that the comment for the function is correctly associated